modules/end-user-guide/pages/mounting-secrets.adoc

```diff
@@ -71,4 +72,12 @@ data:
 ----
 
 When you start a workspace, the `/home/user/.m2/settings.xml` file will be available in the `{devworkspace}` containers.
+
+Maven allows you to set a custom path for the `settings.xml` file.
```

Comment:
```suggestion
With Maven, you can set a custom path for the `settings.xml` file.
```
Anthropomorphism
Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and
emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions.